homeassistant/components/ios/__init__.py

```diff
@@ -240,6 +242,19 @@ async def async_setup(hass, config):
             )
         )
 
+    try:
+        config.get(DOMAIN_MOBILE_APP)[CONF_PUSH]
+    except (TypeError, KeyError):
+        try:
+            conf = config.get(DOMAIN)[CONF_PUSH]
+            _LOGGER.warning(
+                "Deprecitation Warning: Defining push configuration with "
```

Comment:
Should we consider telling them by what HA core version they need to get their migration done with?

Reply:
Yes. We should define it. Would let this sit for one or two version and then remove ios